regexes/device/mobiles.yml

```diff
@@ -414,7 +414,7 @@ CnM:
 
 # RIM/BlackBerry
 RIM:
-  regex: 'BB10;|BlackBerry|rim[0-9]+|PlayBook|STV100-[1234]|STH100-[12]|BBA100-[12]|BBB100-[1234567]|BBC100-1|BBD100-[126]|BBE100-[123456789]|BBF100-[123456789]|BBG100-1|BBH100-1'
+  regex: 'BB10;|BlackBerry|rim[0-9]+|PlayBook|STV100-[1234]|STH100-[12]|BBA100-[12]|BBB100-[1234567]|BBD100-[126]|BB[EF]100-[123456789]|BB[CGH]100-1|BlackBerry.+(?:8520|8530|9000|9220|9300|9320|9360|9630|9720|9800|9860)'
```

Comment:
this is the redundant rule
`BlackBerry.+(<!=\.)(?:8520|8530|9000|9220|9300|9320|9360|9630|9720|9800|9860)` 
we have a rule `|BlackBerry|` which is always executed